ci/release/apply_wheel_modifications.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Usage: bash apply_wheel_modifications.sh <new_version>
+
+VERSION=${1}
+CUDA_SUFFIX=${1}
```

Comment:
This one should be `${2}`, the second argument, right?

Reply:
Yup good catch, thanks.